ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -581,3 +581,52 @@ Set Up Project
     ELSE
         Log    message=Skipping UserWorkloadMonitoring enablement.
     END
+
+Verify OVNS Model Inference Via CLI
+    [Documentation]    Verifies that the inference result of a model is equal to an expected output
+    [Arguments]    ${model_name}    ${inference_input}    ${expected_inference_output}    ${token_auth}=${FALSE}
+    ...    ${test_namespace}=${NONE}    ${kserve_mode}="Serverless"    ${retries}=${10}
+    ${url}=   Set Variable     ${NONE}
+    ${inference_output}=    Set Variable     ${NONE}
+    IF    '${kserve_mode}' == 'Serverless'
+        ${rc}    ${url}=    Run And Return Rc And Output
+        ...    oc get ksvc ${model_name}-predictor -n ${test_namespace} -o jsonpath='{.status.url}'
+         Should Be Equal As Integers    ${rc}    ${0}
+        ${curl_cmd}=   Set Variable    curl -sk ${url}/v2/models/${model_name}/infer -d ${inference_input}
```

Comment:
It will not, the host in the ISVC does not have the protocol in front, see the `Query Model Multiple Times` keyword https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/CLI/ModelServing/llm.resource#L376